src/lib/holocene/menu/menu-button.svelte

```diff
@@ -174,7 +174,7 @@
   }
 
   .table-header {
-    @apply h-auto max-w-fit border-2 border-transparent bg-primary p-0 text-sm text-white focus-visible:border-white;
+    @apply bg-inverse text-white focus-visible:border-white focus-visible:shadow-focus focus-visible:shadow-primary/50;
```

Comment:
Do we want to match the existing style or are there updated designs? 

If we want to just make the existing style compatible with dark mode (e.g. ⬇️)

| Light | Dark |
|--|--|
|<img width="1316" alt="Screenshot 2024-04-09 at 8 00 23 PM" src="https://github.com/temporalio/ui/assets/15069288/bbdbde9f-cf51-4db1-a1fb-f4f004c964ff">| <img width="1311" alt="Screenshot 2024-04-09 at 8 11 46 PM" src="https://github.com/temporalio/ui/assets/15069288/85f54821-3b24-4b37-92f8-1d7501cc6ef6">|

 all we need to do here is to remove the `bg-primary` 

```suggestion
    @apply h-auto max-w-fit border-2 border-transparent p-0 text-sm text-white focus-visible:border-white;
```

and update the `table-header` variant in the `Button` component by replacing `bg-inverse` with `border-transparent` [here](https://github.com/temporalio/ui/blob/main/src/lib/holocene/button.svelte#L45).

Reply:
> and update the table-header variant in the Button component by replacing bg-inverse with border-transparent [here](https://github.com/temporalio/ui/blob/main/src/lib/holocene/button.svelte#L45).

@GraceGardner we'll need this part too in order fix this button ⬇️ 

![Screenshot 2024-04-11 at 9 40 25 AM](https://github.com/temporalio/ui/assets/15069288/e8ec5e8a-7607-48fa-a87a-21a0b1bc8cdb)

Reply:
Thanks for catching this <3